src/status_im/contexts/shell/share/wallet/view.cljs

```diff
@@ -67,25 +67,64 @@
             :on-share-press      #(share-action qr-url share-title)
             :profile-picture     nil
             :unblur-on-android?  true
-            :full-name           (:name account)
-            :customization-color (:color account)
-            :emoji               (:emoji account)
+            :full-name           (.-name account)
+            :customization-color (.-color account)
+            :emoji               (.-emoji account)
             :on-multichain-press #(reset! wallet-type :wallet-multichain)
             :on-legacy-press     #(reset! wallet-type :wallet-legacy)
             :on-settings-press   #(open-preferences @selected-networks)}]]]))))
 
+(defn- indicator
+  [is-active]
+  [rn/view
+   {:style {:width           8
```

Comment:
can we move styles to style file? and also cljs them?
i.e 
`borderRadius` -> `border-radius`
background-color etc.

also I think it's better to use the white from the colors file.
should colors/white

Reply:
updated